README.md

```diff
@@ -11,9 +11,10 @@ Install Docker and add your user to the Docker group (e.g. https://www.digitaloc
 Configure your Docker client to use our registry by following instructions here:
 https://github.com/vimc/montagu-ci#configuring-docker-clients-to-use-the-registry
 
-Start the database by navigating to the `src` folder and running
+Start the dependencies (the database and orderly web) by navigating to the `src` folder and running
```

Comment:
I originally tried to put both the database start and orderly web start into a single gradle task startDependencies, but this caused a big mess on TeamCity because the build environment which runs the unit tests was running that task within a docker container, which was attempting to start the orderly containers-within-a-container with a mounted volume with a path inside the first container.